### ISSUE TITLE
[BUG] `bastille setup` is broken on 14.1

### ISSUE THREAD

**Issue by zilti** [BastilleBSD/bastille#712]
**[MANDATORY] Describe the bug [MANDATORY]**

This is the output of `bastille setup`:

```
bastille_enable: YES -> YES
Configuring bastille0 loopback interface
cloned_interfaces: lo1 bridge1 -> lo1 bridge1
ifconfig_lo1_name: bastille0 -> bastille0
Bringing up new interface: bastille0
ifconfig: interface lo1 already exists
ifconfig: interface bridge1 already exists
Determined default network interface: (vtnet0)
/etc/pf.conf does not exist: creating...
pf_enable: YES -> YES
/usr/sbin/sysrc: cannot create /usr/local/bastille/bastille.conf: No such file or directory
/usr/sbin/sysrc: cannot create /usr/local/bastille/bastille.conf: No such file or directory
```

**[MANDATORY] Bastille and FreeBSD version (paste ``bastille -v && freebsd-version -kru`` output)**
```
0.10.20231125
14.1-RELEASE
14.1-RELEASE
14.1-RELEASE-p2
```

**[MANDATORY] How did you install bastille? (port/pkg/git)**

pkg

**[optional] Steps to reproduce?**

```
# pkg install bastille
# bastille setup
```

**Comment by yaazkal** [comment]
@zilti this has been fixed recently, please try installing Bastille from git. That's gonna be in the next release.

**Comment by yaazkal** [comment]
Just for the record, this two PRs solved the issue: #650 and #709
